decidim-core/app/packs/src/decidim/sticky_header.js

```diff
@@ -3,12 +3,34 @@
 // They increase the discoverability of the elements in the header.
 let prevScroll = window.scrollY;
 const stickyHeader = document.querySelector("[data-sticky-header]");
+const footer = document.querySelector("footer");
+const ctasButtons = document.querySelectorAll("[data-ctas-buttons]");
```

Comment:
As we already have a stickyHeader, I think we should rename this to stickyFooter to keep it consistent. 
```suggestion
const stickyFooter = document.querySelectorAll("[data-sticky-footer]");
```

Reply:
Done